mercury.spec

```diff
@@ -13,6 +13,7 @@ Group:		Development/Libraries
 License:	Aregonee National Laboratory, Department of Energy License
 URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
+Patch:		https://github.com/daos-stack/mercury/daos_8968.patch
```

Comment:
Looks like some kind of whitespace issue here you could fix if you need to push again for some reason.

Reply:
This seems to be github rendering issue; shows up aligned in vim/same whitespaces as lines above

Reply:
Are your tabs set to 4 spaces or something?  I see one tab after `Source0:` but two after `Patch:`.

Reply:
no, it is set to 8. yes there are 2 tabs after patch,  same as 'group' line on line 11 that shows up misaligned in github rendering

Reply:
OK.